src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,100 @@
+package duke;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents the duke.Duke chat-bot, called Nite.
+ * Nite is an interactive app which helps to keep track of tasks.
+ * @author Chia Wen Ling
+ * @version v0.1
+ */
+public class Duke {
+    private static Ui ui;
+    private static Storage storage;
+    private static TaskList tasks;
+    private boolean isInitialised;
+
+    /**
+     * Creates a duke.Duke Chat-bot.
+     *
+     * @param filePath Directory where duke.Duke text file is saved.
+     * @param folderPath Path name of duke.Duke text file to be saved.
+     */
+    public Duke(String filePath, String folderPath) {
+        ui = new Ui();
```

Comment:
i think its better to reference your instance variables starting with the "this" keyword, such as `this.ui` to be clear and prevent variable shadowing